src/three/TilesRenderer.js

```diff
@@ -87,6 +87,12 @@ export class TilesRenderer extends TilesRendererBase {
 	}
 
 	/* Public API */
+	getRootMatrix() {
+
+		return this.root.cached.boxTransform;
+
+	}
+
```

Comment:
Can we change this to use a similar pattern to `getBounds` and pass in a matrix4 to avoid accidental modification of the root box transform instance and return true / false depending on if the root is available. Lets also rename it `getBoundsTransform` so it's clear this and `getBounds` are related:

```js
getBoundsTransform( target : Matrix4 ) : Boolean
```

Reply:
We should also add it to the typescript definitions.